### PR TITLE
add Pixhawk firmware loading info

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: common/source/docs/common-loading-firmware-onto-pixhawk.rst [hunk 1]
```diff
@@ -36,7 +34,7 @@ Install firmware
 ================
 
 In Mission Planner's **SETUP \| Install Firmware** screen
-select the appropriate icon that matches your frame (i.e. Quad, Hexa). 
+select the appropriate icon that matches your vehicle or Multicopter frame type(i.e. Quad, Hexa).
```
Comment (peterbarker) [inline common/source/docs/common-loading-firmware-onto-pixhawk.rst:37]:
```suggestion
select the appropriate icon that matches your vehicle frame type(i.e. Quad, Hexa). 
```

File: common/source/docs/common-loading-firmware-onto-pixhawk.rst [hunk 2]
```diff
@@ -51,6 +49,14 @@ Mission Planner will try to detect which board you are using. It may ask you to
...
    Mission Planner: Install FirmwarePrompt
 
+Often you will be presented with a dropdown box of firmware variants for the board, which you can select from (such as bi-directional DShot variants, if available). For boards which share the Pixhawk board id, the list will be extensive, as shown below:
+
+.. image:: ../../../images/pixhawk-firmware.png
+   :target: ../_images/pixhawk-firmware.png
+
+Select the appropriate firmware for your board. For boards marked "Pixhawk", Pixhawk1 firmware is usually the best choice.
+
+.. warning:: Pixhawk1 boards supplied by non ArduPilot partners (ie. "clones") may not be fully compatible with ArduPilot and loading ArduPilot firmware may result in operational issues.
```
Comment (tridge) [inline common/source/docs/common-loading-firmware-onto-pixhawk.rst:59]: maybe "some boards labelled as Pixhawk 2.4.x may have sensor substitutions which could cause issues. Please see the BARO_OPTIONS parameter for a workaround for a known substitution on some boards of a MS5607 barometer where a MS5611 should be used. IMUs may also be substituted. Where possible please source boards from reputable vendors"

**Review by tridge — CHANGES_REQUESTED** [review]

File: common/source/docs/common-loading-firmware-onto-pixhawk.rst (hunk 2 above)
Comment (rmackay9) [inline common/source/docs/common-loading-firmware-onto-pixhawk.rst:59]: Maybe we can make this more concrete.  For example, "some Pixhawk boards supplied by non ArduPilot partners (ie. "clones") may have unexpected sensor substitutions which may lead to pre-arm checks or no backup IMU".

Comment (rmackay9) [reply common/source/docs/common-loading-firmware-onto-pixhawk.rst:59]: I'd suggest "ArduPilot Parnters" instead of "reputable vendors" maybe.

**Comment by Hwurzburg** [comment]
suggested changes made...merging

**Pull request merged**